### PR TITLE
Stateless statefull update noah

### PR DESCRIPTION
**Pull request by iamNoah1**
### Describe your changes
Follow up for #1731 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/en/stateless-apps.md [hunk 1]
```diff
@@ -1,34 +1,15 @@
...
 ---
 
-A stateless application doesn’t save any client session (state) data on the server where the application lives. 
-Each session is carried out as if it was the first time and responses are not dependent upon data from a previous session and 
-provides functionality to use print services, CDN (Content Delivery Network) or the Web Servers 
-in order to process every short-term request. 
-For example, someone is searching a question in the search engine and pressed the Enter button. 
-In case if the searching operation gets interrupted or closed due to some reason, 
-you have to start a new one as there is no saved data for your previous request.
+Stateless applications process requests as if they were their first.
```
Comment (hlipsig) [inline content/en/stateless-apps.md:8]:
```suggestion
Stateless applications process requests as if each request were the first it's ever been sent. 
```

**Review by hlipsig — CHANGES_REQUESTED** [review]
Suggestions made to improve clarity of language.  No technical issues noted.

**Comment by seokho-son** [comment]
Hi @iamNoah1 
https://github.com/cncf/glossary/pull/2684#pullrequestreview-1769373067 updated the wordlist.txt 
So, this PR does not need to include it. :)

**Review by hlipsig — APPROVED** [review]
LGTM

**Review by nate-double-u — APPROVED** [review]
Thanks for the update, /lgtm

File: content/en/stateful-apps.md [hunk 2]
```diff
@@ -1,29 +1,16 @@
 ---
 title: Stateful Apps
-status: Completed
-category: concept
-tags: ["fundamental", "application", ""]
+status: Feedback Appreciated
```
Comment (iamNoah1) [inline content/en/stateful-apps.md:3]:
```suggestion
status: Completed
```

File: content/en/stateless-apps.md [hunk 3]
```diff
@@ -1,34 +1,15 @@
 ---
-title: Stateless Apps
-status: Feedback Appreciated
-category: technology
-tags: ["fundamental", "application", ""]
+ title: Stateless Apps
+ status: Feedback Appreciated
```
Comment (iamNoah1) [inline content/en/stateless-apps.md:3]:
```suggestion
 status: Completed
```

**Comment by iamNoah1** [comment]
thanks @nate-double-u and @hlipsig for the lgtm, @seokho-son I removed my changes on the wordlist again and merged main.

**Review by seokho-son — APPROVED** [review]
LGTM! Let me merge this PR.

**Pull request merged**